Makefile

```diff
@@ -1,5 +1,6 @@
 test:
 	go test ./...
+	TYPEF91A_ISSUANCE_TEST_VECTORS_IN=typeF91A-issuance-test-vectors-rust.json go test -v -run TestVectorVerifyBatchedPrivateIssuance ./tokens/typeF91A
```

Comment:
Note that the [CI script](https://github.com/cloudflare/pat-go/blob/main/.github/workflows/go.yml#L25) never calls this Makefile.

Reply:
good catch. the action has been updated and is working as expected